WebDriverAgent.xcodeproj/xcshareddata/xcschemes/WebDriverAgentRunner.xcscheme

```diff
@@ -67,7 +67,7 @@
       <EnvironmentVariables>
          <EnvironmentVariable
             key = "USE_PORT"
-            value = "$(USE_PORT)"
+            value = "8100"
```

Comment:
Is it just the default value or you just hardcoded it?

Reply:
i changed it from `$(USE_PORT)` to this. before, it was turning up as `0` anytime i tested the build.

probably there's some way to set the env var from outside xcode but i could not discover it. suggestions welcome.

Reply:
This change breaks Appium's ability to change the WDA port. Another way to do it needs to be found.

Reply:
sorry, forgot this was still included in the merge. will revert.

Reply:
https://github.com/appium/WebDriverAgent/pull/62